tests/test_elasticache.py

```diff
@@ -458,3 +458,34 @@ def test_elasticache_add_security_group(self):
         self.assertEqual(len(resources[0]["SecurityGroups"]), 1)
         self.assertEqual(len(clean_resources[0]["SecurityGroups"]), 2)
         self.assertEqual(len(clean_resources), 3)
+
+
+class TestElastiCacheReplicationGroup(BaseTest):
+
+    def test_elasticache_replication_group(self):
```

Comment:
either this test or the one below is redundant, no need to test the value filter here, the resource query is what you're mainly testing

Reply:
yep, I had to add the second test! Else the PR fails in codecov

Reply:
Removed the extra test, codecov is all good! Don't know what was the problem the other day. Thanks @thisisshi !